### ISSUE TITLE
Be compatible with the Jira bug tracker

### ISSUE THREAD

**Issue by abourdon** [cerberustesting/cerberus-source#1290]
Hi,

Would it possible to be compatible to the Jira bug tracker?

Regards,
Aurelien

**Comment by bcivel** [comment]
Good idea. I think the only change will be to send parameters to Jira using POST (instead of GET for mantis and redmine).

**Comment by abourdon** [comment]
After some investigation, it is possible to create Jira issues via direct HTML link: https://confluence.atlassian.com/jirakb/creating-issues-via-direct-html-links-159474.html

Example:

https://<your_jira_domain>/secure/CreateIssueDetails!init.jspa?pid=11400&issuetype=1&description=this+is+the+description&summary=this+is+the+summary

So issue can be closed because feature already exists in Cerberus

Regards,
Aurélien

**Comment by abourdon** [comment]
Additional information: as explained from the Jira documentation, the `CreateIssueDetails` is quite outdated but still works.

Note there is a Jira ticket about issue pre filling here: https://jira.atlassian.com/browse/JRASERVER-23590

Aurélien

**Comment by apliarchopoulos** [comment]
Hi, would it be possible to implement Jira's REST API ? 

https://developer.atlassian.com/cloud/jira/platform/rest/v3/#api-rest-api-3-issue-post

Alexandre

**Comment by vertigo17** [comment]
Could also be worth with Notion.so
